src/Controller/User/UserTalkController.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Controller\User;
+
+use App\Entity\Talk;
+use App\Form\EditTalkType;
+use App\Form\NewTalkType;
+use App\Repository\TalkRepository;
+use App\UX\UserChartBuilder;
+use Doctrine\ORM\EntityManagerInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Twig\Markup;
+
+class UserTalkController extends AbstractController
+{
+    public function __construct(
+        private UserChartBuilder $userChartBuilder,
+        private TalkRepository $talkRepository,
+        private EntityManagerInterface $em,
+    ) {
+    }
+
+    #[Route('/user/talks', name: 'user_talks')]
+    public function userTalks(Request $request): Response
+    {
+        $talk = new Talk();
+        $form = $this->createForm(NewTalkType::class, $talk);
+        $form->get('users')->setData([$this->getUser()]);
```

Comment:
It would be better to set the data in the talk directly, than passing via the form. Something like:

```
$talk = new Talk();
$task->setUsers([$this->getUser()])
```

With that, you can remove this line, and the one in the template

Reply:
Unfortunately, the user is not set on the talk directly, but on the submit, this is why I couldn't do that

Reply:
Discussed IRL. the type is decoupled from the model. So this is fine 👍🏼